packages/eslint-config-asini/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "eslint-config-asini",
+  "version": "1.0.0",
+  "description": "Shared eslint config for all Asini packages",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"No tests here...\""
```

Comment:
Makes sense.  How's [this](https://github.com/asini/asini/pull/68/files#diff-984b5ef67708f259b7d6a3c5673293e5R3)?